### PR TITLE
Refactor/UI card

### PR DESCRIPTION
**Pull request by RoyAppeldoorn**
## What

Explain what changes inside the code, this can be as simple or complicated as you want as long as it's clear

## Why

Explain why these things are changes. This explanation is for your colleagues and your future self.

## Code Review

Please consider the following checklist when reviewing this Pull Request.  
More background and details [here](https://github.com/kabisa/kabisa-guide/blob/master/organization/process/code-review/README.md).

* [ ] Does the code **actually solve** the problem it was meant to solve?
* [ ] Is the code covered by **unit tests**? **Integration tests**?
* [ ] Does anything here need **documentation**? (Focus on *why*, not *what.*)
* [ ] Does any of this code deal with **privacy sensitive information** or affects **security**? Ask an additional reviewer.
* [ ] Is the code easy to **understand** and **change** in the future?
* [ ] Is the same code or concept **duplicated**? Find a balance between DRYness and readability.
* [ ] Does the code reasonably adhere to the Kabisa [**coding standards**](https://github.com/kabisa/kabisa-guide/blob/master/organization/process/code-review/README.md)?
* [ ] Be kind.

### CODE REVIEW

File: src/ui/Card/Card.module.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 2rem 1.5rem 2rem;
+}
+
+.cardTitle {
+  font-weight: var(--font-weight-bold);
+  line-height: 1;
+  margin-bottom: var(--margin-level-3);
+}
```
Comment (marcianosr) [inline src/ui/Card/Card.module.css:23]: This is not needed right? We can adhere to earlier styles made with Heading component

File: src/ui/Card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+const Card = ({ theme = "light", children }: CardProps) => (
+  <UICard className={classNames(styles.card, styles[theme])}>{children}</UICard>
+);
+
+const CardHeader = ({ children }: { children: ReactNode }) => (
+  <div className={styles.cardHeader}>{children}</div>
+);
+
+const CardTitle = ({ children }: { children: ReactNode }) => (
+  <h1 className={classNames(styles.cardTitle)}>{children}</h1>
```
Comment (marcianosr) [inline src/ui/Card/index.tsx:20]: So this would be a heading component

File: src/ui/Card/Card.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      control: {
+        type: "inline-radio",
+        options: ["light", "dark"],
+      },
+      defaultValue: "light",
+    },
+  },
+};
+
+export const Default: StoryObj<typeof Card> = CardComponent.bind({});
```
Comment (RoyAppeldoorn) [inline src/ui/Card/Card.stories.tsx:39]: **suggestion**: This throws an error and follows the Component Story Format 2, which is the old format. To transform it to [CSF3](https://storybook.js.org/docs/react/api/csf#upgrading-from-csf-2-to-csf-3) you can do the following:

```javascript
const meta: Meta<typeof Card> = {
  component: Card,
  title: "ui/Card",
  args: {
    title: {
      text: "Report",
      iconName: "flag",
    },
    content: (
      <div>
        Banjo-Kazooie is the first game in the Banjo-Kazooie series. It was
        developed by Rare and released in 1998 for the Nintendo 64. In this
        game,
      </div>
    ),
    footer: <footer>Footer content</footer>,
  },
  argTypes: {
    theme: {
      control: {
        type: "inline-radio",
        options: ["light", "dark"],
      },
      defaultValue: "light",
    },
  },
};

export default meta;
type Story = StoryObj<typeof Card>;

export const Default: Story = {
  render: (args) => <Card {...args} />,
};

export const Secondary: Story = {
  args: {
    center: true,
    title: {
      text: "Kudometer",
    },
    date: "13th of May 1991",
    description: "This is a description",
  },
  render: (args) => (
    <div style={{ width: "400px" }}>
      <SecondaryCard {...args} />
    </div>
  ),
};
```

Comment (marcianosr) [reply src/ui/Card/Card.stories.tsx]: Fixed!

**Pull request merged**